.eslintrc

```diff
@@ -1,9 +1,17 @@
 {
+<<<<<<< HEAD
```

Comment:
This merge conflict should not be committed and pushed. Please fix the merge conflict and push a new commit with the fix